q2_diversity/_core_metrics.py

```diff
@@ -57,7 +59,8 @@ def core_metrics_phylogenetic(ctx, table, phylogeny, sampling_depth, metadata,
 
     cr = core_metrics(table=table, sampling_depth=sampling_depth,
                       metadata=metadata, with_replacement=with_replacement,
-                      n_jobs=n_jobs_or_threads)
+                      n_jobs=n_jobs_or_threads,
+                      ignore_missing_samples=ignore_missing_samples)
```

Comment:
Nice! Good find on this one 🙂 this is a great example of how pipelines can depend on other pipelines. Core metrics phylogenetic is just regular core metrics being called plus a few extra methods (which are the ones that use phylogeny as an input).